conf/gulp-tasks/template/bundletemplatestaskfactory.js

```diff
@@ -99,17 +97,26 @@ class BundleTemplatesTaskFactory {
       output: bundleConfig,
       plugins: [
         resolve(),
-        insert.prepend(
-          fs.readFileSync('./conf/gulp-tasks/templates-polyfill-prefix.js').toString(),
-          {
-            include: `./dist/${precompiledFile}`
-          }),
         builtins(),
         commonjs({
           include: './node_modules/**'
         }),
         babel({
-          presets: ['@babel/env']
+          runtimeHelpers: true,
```

Comment:
I don't think we should be adding the polyfills back in

Reply:
I was just making sure that the changes from this commit got in here: https://github.com/yext/answers/pull/1017. Is that not what we want?

Reply:
sorry you're right, totally misread